.github/workflows/matrix.yml

```diff
@@ -73,16 +73,27 @@ jobs:
             tiles: 1
             native: osx
             title: Clang 12, macOS 10.15, Tiles
+          - compiler: g++
+            os: ubuntu-latest
+            cmake: 0
+            tiles: 1
+            title: Mingw-w64 Make cross-compile to Windows with Tiles and Sound
```

Comment:
```suggestion
            title: MinGW-Win64 Cross, Ubuntu 20.04, Tiles
```

Reply:
It does have sound enabled, and I don't think we should be explicitly stating the Ubuntu version since it's just using `ubuntu-latest`.  But I'll fix the Mingw naming; I trust you to know if that's more correct.

Reply:
I have to admit that adding "Ubuntu 20.04" was kind of arbitrary. What I wanted is to make the title to follow a similar style to other build configurations.

Reply:
Good point.  I've rephrased it a bit.